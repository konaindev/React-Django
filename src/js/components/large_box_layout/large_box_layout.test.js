import { LargeBoxLayout } from "./index";
import renderer from "react-test-renderer";

describe("LargeBoxLayout", () => {
  it("renders correctly", () => {
    const props = {
      name: "Test name",
      content: "Test content",
      detail: "This is details",
      detail2: "This is more details",
      innerBox: null
    };
    const tree = renderer.create(<LargeBoxLayout {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
