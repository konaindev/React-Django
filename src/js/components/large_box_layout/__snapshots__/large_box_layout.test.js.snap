// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LargeBoxLayout renders correctly 1`] = `
<div
  className="panel panel--default large-box"
>
  <span
    className="large-box__top-line"
  >
    Test name
  </span>
  <div
    className="large-box__content"
  >
    <span
      className="large-box__content-value"
    >
      Test content
    </span>
  </div>
  <span
    className="large-box__bottom-line"
  >
    This is details
  </span>
  <span
    className="large-box__bottom-line"
  >
    This is more details
  </span>
</div>
`;
